components/script/dom/htmliframeelement.rs

```diff
@@ -67,7 +67,7 @@ pub struct HTMLIFrameElement {
     pipeline_id: Cell<Option<PipelineId>>,
     subpage_id: Cell<Option<SubpageId>>,
     sandbox: MutNullableHeap<JS<DOMTokenList>>,
-    sandbox_allowance: Cell<Option<u8>>,
+    sandbox_allowance: Cell<Option<SandboxAllowance>>,
```

Comment:
Ah I see, this is intended to fix #11599. Please build your branch and fix the compiler errors that result from this change.